src/engine/reflection.h

```diff
@@ -13,7 +13,9 @@
 #define LUMIX_FUNC(F) function<&F>(#F, #F)
 #define LUMIX_CMP(Cmp, Name, Label) cmp<&ReflScene::create##Cmp, &ReflScene::destroy##Cmp>(Name, Label)
 #define LUMIX_PROP(Property, Label) prop<&ReflScene::get##Property, &ReflScene::set##Property>(Label)
+#define LUMIX_READONLY_PROP(Property, Label) prop<&ReflScene::get##Property>(Label)
 #define LUMIX_ENUM_PROP(Property, Label) enum_prop<&ReflScene::get##Property, &ReflScene::set##Property>(Label)
+#define LUMIX_READONLY_ENUM_PROP(Property, Label) enum_prop<&ReflScene::get##Property, &ReflScene::set##Property>(Label)
```

Comment:
you left setter there. Also see the comment few lines up

Reply:
Oops